File: python/ParkingLot.py
# kapasitas
# jumlahKendaraan

from Garage import Garage

class ParkingLot(Garage):
    __kapasitas = ""
    __jumlahKendaraan = ""

    def __init__(self, kapasitas = "", jumlahKendaraan = ""):
        self.__kapasitas = kapasitas
        self.__jumlahKendaraan = jumlahKendaraan

    # getter
        
    def getkapasitas(self):
        return self.__kapasitas
    
    def getjumlahKendaraan(self):
        return self.__jumlahKendaraan
    
    # setter
    
    def setkapasitas(self, kapasitas):
        self.__kapasitas = kapasitas

    def setjumlahKendaraan(self, jumlahKendaraan):
        self.__jumlahKendaraan = jumlahKendaraan

File: c++/ParkingLot.cpp
#pragma once
#include <iostream>
#include <string>
#include "Garage.cpp" // Assuming Garage class is defined in Garage.h

using namespace std;

class ParkingLot : public Garage {
private:
    string kapasitas;
    string jumlahKendaraan;

public:
    ParkingLot(string kapasitas = "", string jumlahKendaraan = "") {
        this->kapasitas = kapasitas;
        this->jumlahKendaraan = jumlahKendaraan;
    }

    string getKapasitas() {
        return kapasitas;
    }

    string getJumlahKendaraan() {
        return jumlahKendaraan;
    }

    void setKapasitas(string kapasitas) {
        this->kapasitas = kapasitas;
    }

    void setJumlahKendaraan(string jumlahKendaraan) {
        this->jumlahKendaraan = jumlahKendaraan;
    }
};
